templates/docs/examples/templates/z-index.html

```diff
@@ -64,8 +64,8 @@
           </ul>
         </li>
       </ul>
-      <ul class="p-navigation__items" role="menu">
-        <li class="p-navigation__item p-subnav" role="menuitem" id="link-1">
+      <ul class="p-navigation__items">
+        <li class="p-navigation__item p-subnav" id="link-2">
```

Comment:
There already is a `link-2` above.

```suggestion
        <li class="p-navigation__item p-subnav" id="link-4">
```